src/components/Router.js

```diff
@@ -90,7 +90,7 @@ export default class Router extends Component {
         div({ style: { marginTop: '1rem' } }, ['Please update your bookmarks to our new URL, firecloud.terra.bio. ' +
         'Welcome to the future of FireCloud!'])
       ]))
-      Nav.history.replace({ search: '' })
+      Nav.history.replace({ search: qs.stringify(_.omit(['fcredir'], qs.parse(Nav.history.location.search, { ignoreQueryPrefix: true, plainObjects: true }))) })
```

Comment:
FYI the toast message won't show up if there is no `#` in the URL. 
@davidangb will firecloud ever redirect to `firecloud.terra.bio` directly? Or will it always redirect to `firecloud.terra.bio/#`?